catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -429,18 +434,31 @@ protected void showRenameDialog(List<T> selectedItems) {
 				.show();
 	}
 
-	protected void showMergeAlert() {
+	protected void showMergeDialog(List<T> selectedItems) {
 		if (adapter.getSelectedItems().size() <= 1) {
 			ToastUtil.showError(getContext(), R.string.am_merge_error);
 		} else {
-			ToastUtil.showError(getContext(), "These two projects cannot yet be merged");
+			TextInputDialog.Builder builder = new TextInputDialog.Builder(getContext());
+
+			builder.setHint(getString(R.string.project_name_label))
+					.setPositiveButton(getString(R.string.ok), (TextInputDialog.OnClickListener) (dialog, textInput)
+							-> {
+						Toast.makeText(getContext(), "Merging " + selectedItems.get(0).getName() + " and "
```

Comment:
This text should be localized

Reply:
May have a look at https://github.com/Catrobat/Catroid/blob/develop/catroid/src/main/res/values/strings.xml to check where you can add strings. Afterwards call them via `R.string.YOUR_STRING_NAME` (if im correct)

Reply:
May have a look at https://github.com/Catrobat/Catroid/blob/develop/catroid/src/main/res/values/strings.xml to check where you can add strings. Afterwards call them via `R.string.YOUR_STRING_NAME` (if im correct)